services/121-service/src/payments/services/payments-creation.service.ts

```diff
@@ -85,45 +81,47 @@ export class PaymentsCreationService {
         );
       }
 
+      await this.paymentsHelperService.checkFspConfigurationsOrThrow(
+        programId,
+        programFspConfigurationNames,
+      );
```

Comment:
this should also go inside createPaymentDryRunPart according to the logic I was aiming for (put all shared code in there)